doc/user/content/security/self-managed/authentication.md

```diff
@@ -26,14 +26,14 @@ used:
 
 Password authentication requires users to log in with a password.
 
-To configure Self-Managed Materialize for password authentication:
+To configure Self-Managed Materialize for password authentication, update the following fields in the [Materialize CR](/installation/appendix-materialize-crd-field-descriptions/):
```

Comment:
So ... let's remove the link here ... because that page just gives all the fields ... and we're giving the specific fields here.

We could go "For all Materialize CR settings, see ... "